x/reward/keeper/grpc_query.go

```diff
@@ -17,3 +17,13 @@ func (k Keeper) Params(c context.Context, req *types.QueryParamsRequest) (*types
 
 	return &types.QueryParamsResponse{Params: params}, nil
 }
+
+// RewardPool queries the community pool coins
+func (k Keeper) Pool(c context.Context, req *types.QueryPoolRequest) (*types.QueryPoolResponse, error) {
+	ctx := sdk.UnwrapSDKContext(c)
+	rewardAcc := k.GetRewardAccount(ctx)
+
+	pool := k.bankKeeper.GetAllBalances(ctx, rewardAcc.GetAddress())
```

Comment:
IMHO, reward keeper implement this query function, and queriers call it, would be better way.

Reply:
https://github.com/xpladev/xpla/pull/18/commits/2542f080f072a4ad4ed025cb7e1d4d7504cc2c94